src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import InfoBadge16 from '../../icon/fill/InfoBadge16';
+// @ts-ignore flow import
+import Toggle from '../../components/toggle';
+import Tooltip from '../../components/tooltip';
+
+import messages from './messages';
+
+import './AdvancedContentInsightsToggle.scss';
+
+interface Props {
+    hasTooltip?: boolean;
+    isActive?: boolean;
```

Comment:
could we rename `isActive` to `isChecked`? this would align the prop name with goal of renaming the `Toggle` prop to checked as well:
https://github.com/box/box-ui-elements/blob/8fd0931f8b336dc23bc24bfe31e0d7401f0a0aaf/src/components/toggle/Toggle.js#L13

Reply:
yeah I think we might be doing too much with the current `type AdvancedContentInsightsType`. I think it would be nice to have them as separate types with the USM flowTypes being:
```
// https://github.com/box/box-ui-elements/blob/master/src/features/unified-share-modal/flowTypes.js

type AdvancedContentInsightsTypes = {
    isAdvancedContentInsightsChecked?: boolean,
    onAdvancedContentInsightsToggle?: Function,
};
```
with these changes: https://github.com/box/box-ui-elements/pull/3116/files#r1046624678